hybrid/pi-monte-carlo.py

```diff
@@ -0,0 +1,14 @@
+import random as r
+#import math as m
```

Comment:
This looks like a line that is no longer needed?

Reply:
Done!